src/functions/template-tags/html.php

```diff
@@ -123,8 +123,72 @@ function tribe_disabled( $disabled, $echo = true ) {
 
 		if ( ! $echo ) {
 			return $disabled_string;
-		} else {
-			echo $disabled_string;
 		}
+
+		echo $disabled_string;
+	}
+}
+
+/**
+ * Adds rel attribute for links.
+ *
+ * @since TBD
+ *
+ * @param string|array $rel  Value(s) for rel attribute.
+ * @param boolean      $echo Whether to echo the string or return it.
+ *
+ * @return string the rel attribute
+ */
+function tribe_rel( $rel, $echo = true ) {
```

Comment:
maybe set a default for `$rel`?

Reply:
There is no default rel value - so if it's not set we don't add the attribute.